### PR TITLE
logviewer: tolerate pyqt4 + python2.7 better

### PR DESCRIPTION
**Pull request by mlyle**
Reported by Troy on IRC.

Siiiiigh.

Tested with full matrix py2 vs py3, pyqt4 vs pyqt5.

### CODE REVIEW

File: python/dronin/logviewer/logviewer.py [hunk 1]
```diff
@@ -163,7 +163,9 @@ def handle_open(ignored=False, fname=None):
     with pg.ProgressDialog("0 objects read...", wait=500, maximum=1000, cancelText=None) as dlg:
         global t
 
-        if not isinstance(fname, six.string_types):
+        print type(fname)
```
Comment (tracernz) [inline python/dronin/logviewer/logviewer.py]: Did you mean this print to be here? This will break with Python3?

Comment (mlyle) [reply python/dronin/logviewer/logviewer.py]: No, i didn't :P

**Comment by mlyle** [comment]
Ugh needs more work, I must have failed the python3 test before.

**Comment by mlyle** [comment]
OK, this is what I'm goin' with. :P

**Pull request merged**